test/unit/test_visual_recognition_v3.py

```diff
@@ -1,233 +1,717 @@
+        return output
+
+    def construct_full_body(self):
+        body = dict()
+        body['classifier_id'] = "string1"
+        return body
+
+    def construct_required_body(self):
+        body = dict()
+        body['classifier_id'] = "string1"
+        return body
+
+
+#-----------------------------------------------------------------------------
+# Test Class for update_classifier
+#-----------------------------------------------------------------------------
+class TestUpdateClassifier():
+
+    #--------------------------------------------------------
+    # Test 1: Send fake data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_update_classifier_response(self):
+        body = self.construct_full_body()
+        response = fake_response_Classifier_json
+        send_request(self, body, response)
+        assert len(responses.calls) == 1
+
+    #--------------------------------------------------------
+    # Test 2: Send only required fake data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_update_classifier_required_response(self):
+        # Check response with required params
+        body = self.construct_required_body()
+        response = fake_response_Classifier_json
+        send_request(self, body, response)
+        assert len(responses.calls) == 1
+
+    #--------------------------------------------------------
+    # Test 3: Send empty data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_update_classifier_empty(self):
+        check_empty_required_params(self, fake_response_Classifier_json)
+        check_missing_required_params(self)
+        assert len(responses.calls) == 0
+
+    #-----------
+    #- Helpers -
+    #-----------
+    def make_url(self, body):
+        endpoint = '/v3/classifiers/{0}'.format(body['classifier_id'])
+        url = '{0}{1}'.format(base_url, endpoint)
+        return url
+
+    def add_mock_response(self, url, response):
         responses.add(responses.POST,
-                      gc_url,
+                      url,
                       body=json.dumps(response),
                       status=200,
                       content_type='application/json')
 
-        vr_service.classify(parameters='{"url": "http://google.com"}')
-
-        vr_service.classify(parameters=json.dumps({'url': 'http://google.com', 'classifier_ids': ['one', 'two', 'three']}))
-        vr_service.classify(parameters=json.dumps({'url': 'http://google.com', 'owners': ['me', 'IBM']}))
-
-        with open(os.path.join(os.path.dirname(__file__), '../../resources/test.jpg'), 'rb') as image_file:
-            vr_service.classify(images_file=image_file)
-        assert len(responses.calls) == 8
-
+    def call_service(self, body):
+        service = VisualRecognitionV3(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['VisualRecognitionV3' may be undefined, or defined from star imports: ibm_watson.visual_recognition_v3](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4835642)